<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariableManagementTab</class>
 <widget class="QWidget" name="VariableManagementTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: white;
}

QPushButton {
    background-color:rgb(154, 217, 234) ;
	color: white;
    min-width: 80px;
	min-height: 40px;
	border-style: solid;
	border-color: rgb(154, 217, 234);
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color:rgb(134, 189, 203) ;
	border-color: rgb(134, 189, 203);
	color: rgb(245, 245, 245);
}

QPushButton:flat {
   background-color:rgb(134, 189, 203) ;
	border-color: rgb(134, 189, 203);
	color: rgb(245, 245, 245);
}

QPushButton:checked {
   background-color:rgb(134, 189, 203) ;
	border-color: rgb(134, 189, 203);
	color: rgb(245, 245, 245);
	
	font: 9pt &quot;Segoe UI&quot;;
	text-decoration: underline;
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item alignment="Qt::AlignTop">
       <widget class="QWidget" name="verticalWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_allvarialbes_header">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>All Variables</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QPushButton" name="btn_open_iv">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid white;
	color: grey;
	font: 12pt &quot;Segoe UI&quot;;
}

QPushButton:hover {
    background-color: white;
    color: dark grey; 
	text-decoration: underline;
}

QPushButton:pressed {
    background-color: white;
}

QPushButton:focus {
    border: 1px solid white;
}

QPushButton::menu-indicator {
    image: none;
}
QPushButton:checked {
	color: black;
	text-decoration: underline;
}</string>
           </property>
           <property name="text">
            <string>Independent Variables (IV)</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QPushButton" name="btn_open_dv">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid white;
	color: grey;
	font: 12pt &quot;Segoe UI&quot;;
}

QPushButton:hover {
    background-color: white;
    color: dark grey; 
	text-decoration: underline;
}

QPushButton:pressed {
    background-color: white;
}

QPushButton:focus {
    border: 1px solid white;
}

QPushButton::menu-indicator {
    image: none;
}
QPushButton:checked {
	color: black;
	text-decoration: underline;
}</string>
           </property>
           <property name="text">
            <string>Dependent Variables (DV)</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QPushButton" name="btn_open_speakers">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid white;
	color: grey;
	font: 12pt &quot;Segoe UI&quot;;
}

QPushButton:hover {
    background-color: white;
    color: dark grey; 
	text-decoration: underline;
}

QPushButton:pressed {
    background-color: white;
}

QPushButton:focus {
    border: 1px solid white;
}

QPushButton::menu-indicator {
    image: none;
}
QPushButton:checked {
	color: black;
	text-decoration: underline;
}</string>
           </property>
           <property name="text">
            <string>Speakers</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QWidget" name="verticalWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QPushButton" name="btn_import_metadata">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Import Metadata
 from file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_export_metadata">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Export Metadata
 from file</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="IV_page">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTreeWidget" name="treeWidget_ivs">
           <property name="styleSheet">
            <string notr="true">QTreeWidget { 
	font-size: 14pt; 
}

QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
	font-size: 24px;
	font: bold
}</string>
           </property>
           <column>
            <property name="text">
             <string>Independent Variables(IV)</string>
            </property>
           </column>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QWidget" name="widget_iv_addedit_container" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="btn_add_iv">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	min-height: 50px;
	max-height:50px;
	min-width: 150px;
	max-width: 150px;
}</string>
              </property>
              <property name="text">
               <string>Add IV</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/images/Plus_Icon.svg</normaloff>:/images/images/Plus_Icon.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_edit_ivs">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	min-height: 50px;
	max-height:50px;
	min-width: 150px;
	max-width: 150px;
}</string>
              </property>
              <property name="text">
               <string>Edit IVs</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/images/edit_icon_white.png</normaloff>:/images/images/edit_icon_white.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="DV_page">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" rowspan="5" colspan="2">
          <widget class="QWidget" name="widget_dvgroup_addnewdvs" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QTreeWidget" name="treeWidget_dvs">
              <property name="minimumSize">
               <size>
                <width>320</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QTreeWidget { 
	font-size: 14pt; 
}

QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
	font-size: 24px;
	font: bold
}</string>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>175</number>
              </attribute>
              <column>
               <property name="text">
                <string>Dependent Variables(DV)</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_dv_addedit_container" native="true">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="0">
                <widget class="QPushButton" name="btn_add_dv">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Add DVs</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/images/images/Plus_Icon.svg</normaloff>:/images/images/Plus_Icon.svg</iconset>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="btn_edit_dvs">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Edit DVs</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/images/images/edit_icon_white.png</normaloff>:/images/images/edit_icon_white.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="3" rowspan="5">
          <widget class="QWidget" name="widget_dvgroup_table" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QWidget" name="widget_dvdetectionheadercontainer" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_detectedvariants">
                 <property name="font">
                  <font>
                   <pointsize>18</pointsize>
                   <bold>true</bold>
                   <underline>true</underline>
                  </font>
                 </property>
                 <property name="text">
                  <string>Detected DV-Variants</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_help">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background-color: white;
    border: 1px solid white;
	color: grey;
}

QPushButton:hover {
    background-color: white;
    color: dark grey; 
	text-decoration: underline;
}

QPushButton:pressed {
    background-color: white;
}

QPushButton:focus {
    border: 1px solid white;
}

QPushButton::menu-indicator {
    image: none;
}</string>
                 </property>
                 <property name="text">
                  <string>Help...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/images/images/Help_Icon.svg</normaloff>:/images/images/Help_Icon.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_unassociated_dvs">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Only show Variants that are not associated to a DV yet?</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget_detectedinformation">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>625</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="rowCount">
               <number>0</number>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>160</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Detected Variant</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Occurences</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Associated DV</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Color</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_corpuswaschanged">
              <property name="text">
               <string>The corpus was changed since the last detection of DV-Variants! Do you wish to automatically detect and extract them again?</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_detectvariants">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Extract DV-Variants from corpus annotations</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_detectednewvariants">
              <property name="text">
               <string>Detected XY new variables!</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2" rowspan="5">
          <widget class="Line" name="line">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Speaker_page">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTreeWidget" name="treeWidget_speakers">
           <property name="styleSheet">
            <string notr="true">QTreeWidget { 
	font-size: 14pt; 
}

QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
	font-size: 24px;
	font: bold
}</string>
           </property>
           <column>
            <property name="text">
             <string>Speakers</string>
            </property>
           </column>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QWidget" name="widget_sp_addedit_container" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="btn_add_speaker">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	min-height: 50px;
	max-height:50px;
	min-width: 150px;
	max-width: 150px;
}</string>
              </property>
              <property name="text">
               <string>Add Speakers</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/images/Plus_Icon.svg</normaloff>:/images/images/Plus_Icon.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_edit_speakers">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	min-height: 50px;
	max-height:50px;
	min-width: 150px;
	max-width: 150px;
}</string>
              </property>
              <property name="text">
               <string>Edit Speakers</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/images/edit_icon_white.png</normaloff>:/images/images/edit_icon_white.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_metadatamanag_savecancel" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btn_save_changes">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: white;
	background-color: rgb(0, 170, 0);
	border-style: solid;
	border-color: rgb(0, 125, 0);
	border-width: 2px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: rgb(0, 134, 0);
}

QPushButton:disabled{
  background-color: rgb(121, 121, 121);
}</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
